src/test/scala/com/databricks/spark/avro/AvroSuite.scala

```diff
@@ -22,6 +22,8 @@ import java.nio.file.Files
 import java.sql.Timestamp
 import java.util.UUID
 
+import org.apache.avro.generic.GenericData.{EnumSymbol, Fixed}
```

Comment:
This import should be grouped with the other Avro imports further down in this file.